objectstore/struct.go

```diff
@@ -0,0 +1,22 @@
+package objectstore
+
+const (
+	Enable     = "enable"
+	VolumeName = "name"
+)
+
+// Objectstore info returns current object store status
+// swagger:model
+type ObjectstoreInfo struct {
```

Comment:
Each of these values should have comments since they are to be consumed by the REST client.

Also, what do you think of making this a message in `api.proto`. Do you think this struct can be used for the SDK? Looks like it could be.

Reply:
>>"Each of these values should have comments since they are to be consumed by the REST client."
Ok

are we going to export all of API i.e in OSD by SDK? If that's true we should move this to api.proto. In fact, I should start working on other API set i.e Schedule Policy, Secrets etc. What do you think?